src/main.ts

```diff
@@ -18,13 +18,13 @@ import {
 } from './nm-service';
 import { GuildServiceMapModel } from './model';
 import { AddCron } from './utility/cron-utility';
-import { FoodCountReminder } from './jobs';
```

Comment:
You don't have to work on this, but you probably shouldn't remove it from the code since I plan to work on it.